server/src/main/java/com/google/coffeehouse/common/StorageHandler.java

```diff
@@ -129,7 +131,7 @@ public static String getClubQuery(DatabaseClient dbClient, String clubId) {
   public static String getBookQuery(DatabaseClient dbClient, String bookId) {
```

Comment:
Is the plan to eventually switch this to return an instance of Book? Instead of a string

Reply:
Yes that is the plan. I'll be doing that in my next PR but Joey needs some fixes in this one so the tests all pass on his end.